lib/hello_web/views/layout_view.ex

```diff
@@ -1,7 +1,8 @@
 defmodule HelloWeb.LayoutView do
   use HelloWeb, :view
 
-  # Phoenix LiveDashboard is available only in development by default,
-  # so we instruct Elixir to not warn if the dashboard route is missing.
-  @compile {:no_warn_undefined, {Routes, :live_dashboard_path, 2}}
+  def title() do
```

Comment:
@YanethM by coding style, the functions with arity 0 should not include parentheses.